docs/source/conf.py

```diff
@@ -55,8 +55,8 @@
 
 # General information about the project.
 project = 'MLflow'
-copyright = 'Databricks 2020. All rights reserved'
-author = 'Databricks'
+copyright = 'MLflow Project a Series of LF Projects, LLC. All rights reserved'
```

Comment:
Is there supposed to be a comma after "Project"?

Reply:
Yeah, I'll include a comma (copy-pasted this directly from externally-provided text, but I'm sure nobody will mind the addition of a comma)